app/templates/settings/password.hbs

```diff
@@ -1 +1 @@
-{{settings/password-section}}
\ No newline at end of file
+{{settings/password-section submit='changePassword' passwordCurrent=passwordCurrent passwordNew=passwordNew passwordRepeat=passwordRepeat isLoading=isLoading}}
```

Comment:
Rename this ... 

`changePassword=(action 'changePassword')` 

You already have another action called `submit` in the component.